app/views/root/_error_page.html.erb

```diff
@@ -26,6 +26,12 @@
       </div>
     </article>
   </div>
+
+<% if status_code %>
+  <script type="text/javascript">
+    GOVUK.httpStatusCode = '<%= status_code %>';
```

Comment:
Could this be just "window.httpStatusCode" instead of GOVUK.httpStatusCode?

(I'm only suggesting this because @edds was suggesting using "window" when we spoke offline.  I'm don't have any particular idea why GOVUK or window might be better.)